src/test/java/uk/gov/hmcts/reform/userprofileapi/repository/AuditRepositoryTest.java

```diff
@@ -35,13 +36,16 @@ public void setUp() {
 
     @Test
     public void findAllTest() {
-        assertThat(auditRepository.findAll().size()).isEqualTo(1);
+        List<Audit> audits = auditRepository.findAll();
+
+        assertThat(audits.size()).isEqualTo(1);
+        assertThat(audits.get(0).getUserProfile()).isEqualTo(userProfile);
```

Comment:
Here along with user profile , you should also check audit table contents.

Reply:
I will add this to the test.

Reply:
This has been done in the latest commit https://github.com/hmcts/rd-user-profile-api/pull/240/commits/d027c3097f149f78fa40021e6267a2fa3ff5c214 & https://github.com/hmcts/rd-user-profile-api/pull/240/commits/b6c34b5ddaf424f64e598d595a70435063aeaa72